app/components/configurationStages/personalData.tsx

```diff
@@ -1,31 +1,33 @@
-import React from 'react';
-import styles from '../Configuration.scss';
-import { EMPLOYMENT_STATUS, SALUTATIONS, USEACCOUNT } from '../../reducers/configuration';
-import type { Configuration } from '../../reducers/configuration';
-import type { InheritedProps, StageDescription } from './types';
-import Disclaimer from './disclaimer';
-import TextField from '../inputs/TextInput';
-import NumberField from '../inputs/NumberField';
-import EnumField from '../inputs/EnumField';
-import YesNo from '../inputs/YesNo';
-
-const twoTextFieldsInLineStyles = { width: '200px' };
-
+import React from "react";
+import styles from "../Configuration.scss";
+import { EMPLOYMENT_STATUS, SALUTATIONS, USEACCOUNT } from "../../reducers/configuration";
+import type { Configuration } from "../../reducers/configuration";
+import type { InheritedProps, StageDescription } from "./types";
+import Disclaimer from "./disclaimer";
+import TextField from "../inputs/TextInput";
+import NumberField from "../inputs/NumberField";
+import EnumField from "../inputs/EnumField";
+import YesNo from "../inputs/YesNo";
```

Comment:
confused that the formatter *removed* the newline after imports

Reply:
yes, that's the flow->ts formatter that did such things